schema.sql

```diff
@@ -0,0 +1,9 @@
+
+/* Database schema to keep the structure of entire database. */
+CREATE TABLE animals(id integer NOT NULL default nextval('seq_id'),
+ name varchar(100),
+ date_of_birth date,
+ escape_attempts int,
+ neutered boolean,
+ weight_kg float,
```

Comment:
- **[OPTIONAL]** Kindly change float by *decimal* to avoid errors in your queries. 💡 The choice between FLOAT and DECIMAL in PostgreSQL depends on your specific needs. If you require exact precision in your calculations, it's preferable to use DECIMAL. If precision is not critical, and you want to save storage space, FLOAT may be suitable.